---
bool success
float64 min_position
float64 max_position
